### DISCUSSION TITLE
citation paper for citing JBrowse2

### DISCUSSION THREAD

**Discussion by Sotdo** [GMOD/jbrowse-components#2668] [Q&A] [answered]
Hello,

I wanna know which paper or papers should I cite if I am using JBrowse2 for visualizing mapping results in my publication?

thanks

**Comment by cmdcolin** [comment, marked as answer]
We don't yet have a paper to cite for jbrowse 2 that covers everything yet, we are developing a manuscript as we speak though! If it is helpful you can cite our older jbrowse paper or our JBrowseR paper which uses jbrowse 2

**Reply by Sotdo** [reply]
OK. Thanks.
